### PR TITLE
[docs]: RBD and Nomad integration

### PR DESCRIPTION
**Pull request by emenguy**
The goal of this PR is to add documentation for Nomad, similar to the one that already exists for Kubernetes.
https://docs.ceph.com/en/latest/rbd/rbd-kubernetes/

### CODE REVIEW

**Comment by emenguy** [comment]
It's just a draft for the moment, I need to tests all steps.

**Comment by zdover23** [comment]
I'll wait until all the steps are tested to perform the grammar and elegance review.

**Comment by emenguy** [comment]
I tested instructions using the output, it perfectly works.

File: doc/rbd/rbd-nomad.rst [hunk 1]
```diff
@@ -0,0 +1,431 @@
+==============================
+ Block Devices and Nomad
+==============================
+
+Like Kubernetes, Nomad can use Ceph Block Device thanks to `ceph-csi`_, 
+which allow to dinamacially provision RBD images or import existing one.
```
Comment (mykaul) [inline doc/rbd/rbd-nomad.rst:6]:
```suggestion
which allow to dynamically provision RBD images or import existing one.
```

File: doc/rbd/rbd-nomad.rst [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+
+Nomad must have `rbd` module loaded, check if it's the case.::
+
+        $ lsmod |grep rbd
+        rbd                    94208  2
+        libceph               364544  1 rbd
+
+If it's not the case, load it.::
+
+        $ modprobe rbd
```
Comment (runsisi) [inline doc/rbd/rbd-nomad.rst:105]: needs `sudo`?

Comment (emenguy) [reply doc/rbd/rbd-nomad.rst]: Yep you are right, thanks.

Fixed

File: doc/rbd/rbd-nomad.rst (hunk 1 above)
Comment (emenguy) [reply doc/rbd/rbd-nomad.rst]: Thanks!
Fixed

**Review by zdover23 — APPROVED** [review]

**Pull request merged**